### ISSUE TITLE
Validate file format

### ISSUE THREAD

**Issue by ppeczek** [mafintosh/playback#114]
Right now, you can load any kind of files: documents, images etc. Playback then doesn't display them and stops at black frame. This behaviour looks like playback was suspended. Really needs file format validation!

![screenshot from 2016-06-14 12 50 30](https://cloud.githubusercontent.com/assets/2931838/16040381/6c457fbc-322f-11e6-87f8-a1acfddfade8.png)
![screenshot from 2016-06-14 12 57 34](https://cloud.githubusercontent.com/assets/2931838/16040419/b1933ce4-322f-11e6-8eca-5d9261fb6665.png)

**Comment by freeall** [comment]
That's a good idea. Is it something you would want to try to do @ppeczek?

**Comment by ppeczek** [comment]
Yes, but let's talk about it. My idea is quite naive: textfile with list of
supported extensions, splitting filename by ".", checking if extension is
supported
14 cze 2016 13:01 "Tobias Baunbæk" notifications@github.com napisał(a):

**Comment by mafintosh** [comment]
a module that does this would be cool (if it doesn't already exist)

**Comment by ppeczek** [comment]
Ok then, give me few hours and I will take care of this matter.

**Comment by ppeczek** [comment]
https://github.com/electron/electron/blob/master/docs/api/dialog.md
I found it but just for dialog. It has nothing to do with validation. I've already written part of code for extensions. I have to do sanitizer and we're home
